app/controllers/FallbackFailureController.scala

```diff
@@ -17,24 +17,28 @@
 package controllers
 
 import handlers.ErrorHandler
+
 import javax.inject.Inject
 import play.api.Logging
 import play.api.i18n.I18nSupport
 import play.api.mvc.{Action, AnyContent, MessagesControllerComponents}
 import uk.gov.hmrc.play.bootstrap.frontend.controller.FrontendBaseController
 import utils.Session
 
+import scala.concurrent.{ExecutionContext, Future}
+
 class FallbackFailureController @Inject()(
                                            val controllerComponents: MessagesControllerComponents,
                                            errorHandler: ErrorHandler
-                                         )
+                                         )(implicit val ec: ExecutionContext)
   extends FrontendBaseController
     with I18nSupport
     with Logging {
 
-  def onPageLoad(): Action[AnyContent] = Action {
+  def onPageLoad(): Action[AnyContent] = Action.async {
     implicit request =>
-      logger.error(s"[Verifying][Trust IV][Session ID: ${Session.id(hc)}] Trust IV encountered a problem that could not be recovered from")
-      InternalServerError(errorHandler.internalServerErrorTemplate)
```

Comment:
Is the use of `Future.successful` here definitely correct for an internal server error?